### PR TITLE
chore(sysdig, node-analyzer): bump sysdig/vuln-runtime-scanner to v1.5

### PR DESCRIPTION
**Pull request by michele-mangili**
## What this PR does / why we need it:

Extended Changelog:

* Runtimescanner bumped to 1.5
     * Updated the analyzer to inspects also vendor paths
     * Added support to apply image based accepts for all versions of the image, that image in a specific registry & repository, as well as a contain string for customized subsets of the environment

## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] Title of the PR starts with type and scope, (e.g. feat(agent,node-analyzer,sysdig-deploy):)
- [ ] Chart Version bumped for the respective charts
- [ ] Variables are documented in the README.md (or README.tpl in some charts)
- [ ] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [ ] All test files are added in the tests folder of their respective chart and have a "_test" suffix

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

File: charts/node-analyzer/values.yaml [hunk 1]
```diff
@@ -264,7 +264,7 @@ nodeAnalyzer:
     probesPort: 7002
     image:
       repository: sysdig/vuln-runtime-scanner
-      tag: 1.4.12
+      tag: 1.5
```
Comment (michele-mangili) [inline charts/node-analyzer/values.yaml]: Not sure if this may be treated as a float by helm.

Comment (ProvoK) [reply charts/node-analyzer/values.yaml]: I don't think it will, but if you want to be sure quote it with `"`

**Review by ProvoK — APPROVED** [review]
LGTM, but please remember to add the extended changelog in the commit body.

Example: https://github.com/sysdiglabs/charts/commit/a8daf9cb5776d1b154e4dfbf54a551c85bec1bcb

File: charts/node-analyzer/values.yaml (hunk 1 above)
Comment (michele-mangili) [reply charts/node-analyzer/values.yaml]: Better safe than sorry, I'll update this.

**Review by mech-pig — APPROVED** [review]

**Pull request merged**